recipe/install-py-xgboost.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 pushd ${SRC_DIR}/python-package
-  ${PYTHON} setup.py install --use-system-libxgboost --single-version-externally-managed --record=record.txt
+  ${PYTHON} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
We should keep `setup.py install` for version 1.7.x, since it doesn't yet use the new PEP 517 method of installation.